tests/test_event.py

```diff
@@ -28,6 +102,65 @@ def test_eventview_without_id(self, client):
         response = client.get("/event/view")
         assert response.status_code == HTTP_404_NOT_FOUND
 
+    def test_eventedit_missing_old_invites(self, client, user):
+        response = client.post("/event/edit",
+                               data=CORRECT_EVENT_FORM_DATA)
+        assert response.status_code == HTTP_302_FOUND
+
+        different_invitees_event = CORRECT_EVENT_FORM_DATA.copy()
+        different_invitees_event['invited'] = 'c@c.com,d@d.com'
+        response = client.post("/event/edit",
+                               data=different_invitees_event)
+        assert response.status_code == HTTP_302_FOUND
+        for invitee in CORRECT_EVENT_FORM_DATA["invited"].split(","):
+            assert invitee in response.headers['location']
+
+    def test_eventedit_bad_emails(self, client, user):
+        response = client.post("/event/edit",
+                               data=BAD_EMAILS_FORM_DATA)
+        assert response.status_code == HTTP_302_FOUND
+
+        different_invitees_event = CORRECT_EVENT_FORM_DATA.copy()
+        different_invitees_event['invited'] = 'c@c.com,d@d.com'
+        response = client.post("/event/edit",
+                               data=different_invitees_event)
+        assert response.status_code == HTTP_302_FOUND
+        for invitee in CORRECT_EVENT_FORM_DATA["invited"].split(","):
+            assert invitee in response.headers['location']
+        assert 'ccc' not in response.headers['location']
+
+    def test_eventedit_post_correct(self, client, user):
+        response = client.post("/event/edit",
+                               data=CORRECT_EVENT_FORM_DATA)
+
+        assert response.status_code == HTTP_302_FOUND
+        assert client.app.url_path_for('eventview', id=1).strip('1') in \
+               response.headers['location']
+
+    def test_eventedit_post_wrong(self, client, user):
+        response = client.post("/event/edit",
+                               data=WRONG_EVENT_FORM_DATA)
+        assert response.json()['detail'] == 'VC type with no valid zoom link'
+
+    def test_eventedit_with_pattern(self, client, user):
+        response = client.post("/event/edit",
+                               data=CORRECT_EVENT_FORM_DATA)
+        assert response.status_code == HTTP_302_FOUND
+
+        response = client.post("/event/edit",
+                               data=WEEK_LATER_EVENT_FORM_DATA)
+        assert response.status_code == HTTP_302_FOUND
+        assert 'Same event happened 1 weeks before too. ' in \
+               response.headers['location'].replace('+', ' ')
+
+        response = client.post("/event/edit",
+                               data=TWO_WEEKS_LATER_EVENT_FORM_DATA)
+        assert response.status_code == HTTP_302_FOUND
+        assert 'Same event happened 1 weeks before too. ' in \
```

Comment:
Please use parenthesis instead using `\` (search in other places in the code too)